### ISSUE TITLE
Test if index has parity with ord for satranges

### ISSUE THREAD

**Issue by sandshrewmetadev** [sandshrewmetaprotocols/metashrew-ord#2]
Before attempting to snapshot satranges up to the inscription genesis block, it is necessary to test and confirm with decent confidence that the set of satranges is correct for all unspent outpoints at a block height where we can expect the complexity of sat partitions provides us a decent level of coverage if we compare to ord as a sanity check.

We would like to be able to choose any individual sat number as it relates to ordinal theory and know what outpoint that sat exists on.

**Comment by hans-crypto** [comment]
So what is the status of this? Can you confirm that you now produce the exact same sat numbers as ord? 🙂

**Comment by kungfuflex** [comment]
Yes this implementation does correctly produce the same satranges as ord, is what we found. Closing this.

Further work will be to port this logic to Rust since we now have a very thorough binding and support framework for architecting metashrew programs in Rust, relative to our assemblyscript environment.
